src/fuzz_tests.zig

```diff
@@ -103,5 +95,26 @@ fn smoke() !void {
         }
     }
 
-    log.info("done {}", .{std.fmt.fmtDuration(timer_all.lap())});
+    log.info("done in {}", .{std.fmt.fmtDuration(timer_all.lap())});
+}
+
+fn main_single(cli_args: CliArgs) !void {
+    assert(cli_args.positional.fuzzer != .smoke);
+
+    const seed: usize = cli_args.seed orelse seed: {
+        // If no seed was given, use a random seed instead.
+        var seed_random: u64 = undefined;
+        try std.os.getrandom(std.mem.asBytes(&seed_random));
+        break :seed seed_random;
+    };
+    log.info("Fuzz seed = {}", .{seed});
+
+    var timer = try std.time.Timer.start();
+    switch (cli_args.positional.fuzzer) {
+        .smoke => unreachable,
+        inline else => |fuzzer| try @field(Fuzzers, @tagName(fuzzer)).main(
```

Comment:
I tried `const fuzz_main: fn(fuzz.FuzzArgs) !void = switch(...)` version, but that doesn't work: an `!void` is an _inferred_ error union, it can't be used with function pointers.